extensions/blocks/opentable/opentable.php

```diff
@@ -12,52 +12,28 @@
 const FEATURE_NAME = 'opentable';
 const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
 
-/**
- * Check if the block should be available on the site.
- *
- * @return bool
- */
-function is_available() {
-	if (
-		defined( 'IS_WPCOM' )
-		&& IS_WPCOM
-		&& function_exists( 'has_any_blog_stickers' )
-	) {
-		if ( has_any_blog_stickers(
-			array( 'premium-plan', 'business-plan', 'ecommerce-plan' ),
-			get_current_blog_id()
-		) ) {
-			return true;
-		}
-		return false;
-	}
-
-	return true;
-}
 
 /**
  * Registers the block for use in Gutenberg
  * This is done via an action so that we can disable
  * registration if we need to.
  */
 function register_block() {
-	if ( is_available() ) {
-		jetpack_register_block(
-			BLOCK_NAME,
-			array( 'render_callback' => 'Jetpack\OpenTable_Block\load_assets' )
-		);
-	} else {
-		\Jetpack_Gutenberg::set_extension_unavailable(
-			BLOCK_NAME,
-			'missing_plan',
-			array(
-				'required_feature' => 'opentable',
-				'required_plan'    => 'premium-plan',
-			)
-		);
-	}
+	jetpack_register_premium_wpcom_block(
```

Comment:
Could the plan requirement information be passed to `jetpack_register_block` instead of a new separate function?

I don't have strong opionion on this but just seems easier to maintain one function.

Reply:
It also feels little confusing to have "wpcom" in the registration name, isn't this block available for Jetpack sites?